src/components/questions/OpenTextArea.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { StyleSheet, TextInput, TextInputProps } from 'react-native';
+import FONTS from '../../assets/fonts';
+
+const OpenTextArea = (props: TextInputProps) => {
```

Comment:
Weet niet of het van toepassing is, maar hoe krijgen we hier de tekst weer uit, bij buttons hebben we een onClicked methode die wordt meegegeven, weet niet of hier iets vergelijkbaars voor is?

Reply:
als het goed is gwn lege parameter 'const OpenTextArea = () => {'

Reply:
weet niet precies wat je via TextInputProps wil meegeven

Reply:
Oh goed dat je het zegt, er zit een `onTextChange()` op de TextInput. Deze zou dan inderdaad een callback moeten ontvangen vanuit de parent.

Reply:
nvm, hier op terugkomend, Je kan dit doen in de parent `<OpenTextArea onTextChange={(value)=> {}}/>`. De props worden doorgegeven naar de TextInput zelf

Reply:
Oh @IntissarJ de TextInputProps zijn alle standaard props van een TextInput. Ik wil alle standaard TextInput props doorgeven naar mijn custom TextInput. Hierdoor kun je dus de `<OpenTextArea onTextChange={(value)=> {}}/>` aanroepen bijvoorbeeld

Reply:
Ah ik snap het! Ik had ook de onTextChange niet opgevallen. Dankje voor uitleg @wwessel01

Reply:
Ah oke top!